File: B8_RestFragmentsAsync/WeatherReportApp/app/src/main/java/com/example/tsvetelinastoyanova/weatherreportapp/models/Wind.java

package com.example.tsvetelinastoyanova.weatherreportapp.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Wind {

    @SerializedName("speed")
    @Expose
    private double speed;

    public double getSpeed() {
        return speed;
    }

    public void setSpeed(double speed) {
        this.speed = speed;
    }

}

File: I1_MVPAndRetrofit_Kotlin/app/src/main/java/com/example/tsvetelinastoyanova/weatherapp/model/currentweather/Wind.kt
package com.example.tsvetelinastoyanova.weatherapp.model.currentweather

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Wind(
        @SerializedName("speed") val speed: Double,
        @SerializedName("deg") val deg: Int
) : Parcelable